src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexInterpreter.cs

```diff
@@ -12,14 +12,21 @@ namespace System.Text.RegularExpressions
     internal sealed class RegexInterpreterFactory : RegexRunnerFactory
     {
         private readonly RegexInterpreterCode _code;
+        private readonly CultureInfo? _culture;
```

Comment:
Should this store TextInfo instead rather than getting it from the culture whenever a new interpreter is created?